Geometry/TrackerGeometryBuilder/test/python/testTrackerHierarchy_cfg.py

```diff
@@ -1,10 +1,14 @@
 import FWCore.ParameterSet.Config as cms
 
-process = cms.Process("GeometryTest")
+from Configuration.Eras.Era_Phase2C17I13M9_cff import Phase2C17I13M9
```

Comment:
there's already a file that does this:

https://github.com/cms-sw/cmssw/blob/c312b85e0474fb1c058f8e2c8e0aadcf6b69c30d/Geometry/TrackerGeometryBuilder/test/python/testPhase2TrackerHierarchy_cfg.py#L19-L29

introduced in https://github.com/cms-sw/cmssw/pull/45866. I would revert the changes to this file (and keep it as reference for the Run3 geometry)

Reply:
change reverted and branch updated